learners/nonuniform_quantization/learner.py

```diff
@@ -255,10 +255,21 @@ def __build_train(self):
           if v not in clusters]
 
       # determine the var_list optimize
-      if FLAGS.nuql_opt_mode == 'both':
-        optimizable_vars = self.trainable_vars
-      elif FLAGS.nuql_opt_mode == 'clusters':
-        optimizable_vars = clusters
+      # Temperally use GradientDescentOptimizer for fintune in during rl-rollouts. This fixes issue #76. 
+      # However, this will make the fintuning result during rl-rollouts based on GD rather then Adam.
+      # After the best policy is figured out, final quantizing is still with Adam.
+      # TODO:find a better way solving the problem
+      if FLAGS.nuql_opt_mode != 'weights':
+        if FLAGS.nuql_opt_mode == 'both':
+          optimizable_vars = self.trainable_vars
+        else:
+          optimizable_vars = clusters
+        if  FLAGS.nuql_enbl_rl_agent == True:  
```

Comment:
There are two whitespaces between `if` and `FLAGS.nuql_enbl_rl_agent`. Please reduce to one.

Reply:
Also, change the `if` condition to `if FLAGS.nuql_enbl_rl_agent` (remove "== True"), since `FLAGS.nuql_enbl_rl_agent` is boolean.